### PR TITLE
Add an initial test and infrastructure to support

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
There's a few things going on here. My goal was to add one initial working actions test to set the stage for further tests being added.

File: src/store/index.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ export function init (...middleware) {
         ...middleware
     ]);
 
-    registry.register({ compareReducer });
+    registry.register({
+        compareReducer: reducers.compareReducer,
+        statusReducer: reducers.statusReducer
```
Comment (ehelms) [inline src/store/index.js]: @johnsonm325 This is the reducer split we discussed

**Comment by beav** [comment]
how do you run the tests?

**Comment by ehelms** [comment]
`npm run test`

**Comment by ehelms** [comment]
This already handles running tests -- https://travis-ci.org/RedHatInsights/drift-frontend/builds/489090917#L572

**Pull request merged**